index.html

```diff
@@ -843,6 +843,64 @@ <h2>Skills</h2>
    
     </section>
     <!--about me section ends here -->
+     
+              
+    <!--contact me   -->
+    <section class="contact">
+
+      <div class="contact-container">
+
+          <div class="contact-container-desktop">
+           
+            <h2 id="contact">Contact me</h2>
+            
+            <p class="contact-P contact-P-desktop">
+              If you have an application you are interested in developing, a feature that you need built or a project that needs coding. 
+              I’d love to help with it!
+            </p>
+
+              <!--  contact form section  -->
+              <form class="contact-form" action="https://formspree.io/f/xaygnnvv" method="POST">
+                
+                <ul class="form-section">
+
+                  <li>
+                    
+                  <input type="text" name="name" class="input-desktop" maxlength="30" placeholder="Enter full name"
+                      required>
+  
+                  </li>
+  
+                  <li>
+
+                  <input type="email" name="email" class="input-desktop" placeholder="Enter your email  " required>
+  
+                  </li>
+  
+                  <li>
+                    
+                    <textarea name="message" cols="43" rows="9" maxlength="500" placeholder="Write your message here"
+                      required class="input-desktop"></textarea>
+  
+                  </li>
+  
+                </ul>
+  
+                <button type="submit" class="getInTouch getInTouch-desktop">Get in Touch</button>
+  
+              </form>
+               <!-- end contact form -->
+              <div class="home-indicator">
+
+                <img src="./assets/contact/home-indicator-shape.jpg" class="mobile-footer-image"/>
+              
+              </div>
+              
+            </div>
+        
+          </div>
+    </section>
```

Comment:
[Optional] 👍📱 Excellent work, your code appears almost identical on the 375px screen size! However, there are some improvements needed to meet project requirements for mobile responsiveness:

- [ ] [Optional]The "Contact Us" section needs to be responsive for mobile devices of different screen sizes. It should work well from a minimum mobile screen size of 375px, and errors in screen sizes less than 375px can be ignored. Furthermore, ensure the mobile version remains responsive until the desktop screen size of 768px.

[mobile-first-portfolio.webm](https://github.com/melkamu372/Portfolio-mobile-first/assets/55840999/6951ecdd-5872-4aaf-b1e6-8fbe0b2c8a87)

- [ ] [Optional] Instead of using fixed width and height for most elements, it is recommended to utilize responsive values in your CSS rules, such as percentages (e.g., width: 80%), rather than pixels. This will ensure that your layout adapts smoothly to various screen sizes.
![image](https://github.com/shimels1/portfolio_microverse/assets/55840999/8b7488e3-b3f1-46ed-8524-a6ccf168343e)

- [ ] [Optional] Moreover, you have a margin of 2 rem = 32px in the left-right side. But as per design, it should be 24px. please follow the design and update your code to match it. 🙏 
![image](https://github.com/shimels1/portfolio_microverse/assets/55840999/0f5c8962-9fe1-49b0-b97e-d33d42e2c9e2)

By addressing these points, your "Contact Us" section will be more responsive and visually aligned with the project requirements. Keep up the great work, and I look forward to seeing the improvements! 🚀📱🎨